ci/scripts/compute-node-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+echo "--- Install llvm-tools-preview, clippyllvm-cov, nextest"
+rustup component add llvm-tools-preview clippy
+
+cargo install cargo-llvm-cov
```

Comment:
Should be installed in Dockerfile instead of here.

Reply:
ok, I will remake a full image later.